docker/vars.opentest.sh

```diff
@@ -2,4 +2,9 @@
 
 export GPC_CONSUMER_SERVER_PORT="8080"
 export GPC_CONSUMER_GPC_GET_URL="https://messagingportal.opentest.hscic.gov.uk:19192"
-export GPC_CONSUMER_GPC_STRUCTURED_PATH="/B82617/STU3/1/gpconnect/fhir/Patient/\$gpc.getstructuredrecord"
\ No newline at end of file
+export GPC_CONSUMER_GPC_STRUCTURED_PATH="/B82617/STU3/1/gpconnect/fhir/Patient/\$gpc.getstructuredrecord"
```

Comment:
It should be. I do not see a reason why github highlighted it.

Reply:
Github probably do not have very specific highlighting for bash. Literal $ needs to be escaped \$ otherwise the interpreter treats $gpc as a variable. An alternate approach is to use single quotes.

Reply:
Yes, it breaks the tests on jenkins. I consulted it with @LukaszKwasniewski and this is the right approach.